gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildFullBundleTask.java

```diff
@@ -79,21 +93,25 @@ DirectoryProperty getConfigFolder() {
     public void perform() {
         final FullBundleCreator fullBundleCreator = getInstance(FullBundleCreator.class);
         final String bundleDirectory = into.getAsFile().get().getPath();
-        final List<File> metaDataFiles = collectFiles(bundleDirectory, YML_EXTENSION);
+        final List<File> metaDataFiles = collectFiles(bundleDirectory, JsonFileUtils.METADATA_FILE_NAME_SUFFIX);
         if(metaDataFiles.isEmpty()) {
             throw new MissingEnvironmentException("Metadata file does not exist.");
         }
+
         metaDataFiles.stream().forEach(metaDataFile-> {
-            final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configFolder.getAsFile().get());
+            final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configFolder.getAsFile().getOrNull());
             if (null != bundleEnvironmentValues) {
-                final String bundleFileName = bundleEnvironmentValues.getLeft() + "." + configName.get() + ".full.bundle";
+                final String bundleFileName = bundleEnvironmentValues.getLeft() + "-full.install.bundle";
```

Comment:
Please revert this change. It will be a conflict with #234

Reply:
@nikhilahuja , I will take changes from your PR